config/config.go

```diff
@@ -994,7 +994,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.sa_lunamedia.endpoint", "http://balancer.lmgssp.com/pserver")
 	v.SetDefault("adapters.madvertise.endpoint", "https://mobile.mng-ads.com/bidrequest{{.ZoneID}}")
 	v.SetDefault("adapters.marsmedia.endpoint", "https://bid306.rtbsrv.com/bidder/?bid=f3xtet")
-	v.SetDefault("adapters.mediafuse.endpoint", "http://ghb.hbmp.mediafuse.com/pbs/ortb")
+	v.SetDefault("adapters.mediafuse.endpoint", "http://ib.adnxs.com/openrtb2")
```

Comment:
Switches to the AppNexus endpoint. It looks like Mediafuse had their own endpoint previously, is that still the case?

Reply:
We don't have an end point, let it be pointing got appnexus

Reply:
We don't have an end point, let it be pointing got appnexus